src/components/SingleSelectOption.js

```diff
@@ -28,6 +28,7 @@ export default class SingleSelectOption extends Component {
           rrs__option
           ${(singleSelectSelectedIndex === index) ? 'rrs__option--selected' : ''}
           ${(nextPotentialSelectionIndex === index) ? 'rrs__option--next-selection' : ''}
+          ${(option.disabled === true) ? 'rrs__option--disabled' : ''}
```

Comment:
This <li> needs an aria prop now I reckon `aria-disabled={option.disabled ? 'true' : 'false'}`